orchard/orchard_runner.go

```diff
@@ -79,20 +79,22 @@ func s3ArtifactGenerate(artifact string, command string) (string, error) {
 
 func processCmd(command string, pwd string) (string, error) {
 	if err := os.Chdir(pwd); err != nil {
-		return "", fmt.Errorf("cd %v has error:%w", pwd, err)
+		return "", fmt.Errorf("cd %v has error: %w", pwd, err)
 	}
 	cmds, err := parseCommandLine(command)
 	if err != nil {
-		return "", fmt.Errorf("parse command line error%w", err)
+		return "", fmt.Errorf("parse command line error %w", err)
 	}
 	if len(cmds) < 1 {
 		return "", fmt.Errorf("Invalid command line %s", command)
 	}
-	out, err := exec.Command(cmds[0], cmds[1:]...).Output()
+	cmd := exec.Command(cmds[0], cmds[1:]...)
+	out, err := cmd.CombinedOutput()
+	output := string(out)
```

Comment:
maybe swap L93 with the err check.  If err, not call `string(out)`

Reply:
@schowsf the `string(out)` is now needed no matter error or not, when error, it contains the error stack, if not, it contains the json payload output